### PR TITLE
validate Terraform config in CI

### PR DESCRIPTION
**Pull request by afeld**
Baby step towards https://github.com/18F/tts-tech-portfolio/issues/443.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+  terraform: circleci/terraform@2.0.0
 workflows:
   node-tests:
     jobs:
       - node/test:
           app-dir: inspector
+  terraform:
+    jobs:
+      - terraform/validate:
+          backend: false
```
Comment (afeld) [inline .circleci/config.yml:13]: No credentials required.

**Review by JJediny — APPROVED** [review]
differing here not sure the value as Im not seeing a check in check?

**Comment by afeld** [comment]
It's doing a `terraform validate` — can see the run from this pull request in https://app.circleci.com/pipelines/github/18F/tts-tech-portfolio/188/workflows/a971e0fa-25db-4208-b038-d42892fcf5fb/jobs/189.

**Comment by afeld** [comment]
It's basically just protecting against syntax errors at this point, but better than nothing.

**Pull request merged**